providers/mdxProvider.tsx

```diff
@@ -1,10 +1,11 @@
 import { useEffect } from 'react';
 import { MDXProvider as BaseMDXProvider } from '@mdx-js/react';
+import { MDXRemote } from 'next-mdx-remote';
 import HighlightJS from 'highlight.js/lib/core';
 import HighlightJavaScript from 'highlight.js/lib/languages/javascript';
 import AnchoredHeading from '../components/AnchoredHeading';
 import NodeApiVersionLinks from '../components/Docs/NodeApiVersionLinks';
-import type { FC, PropsWithChildren } from 'react';
+import type { FC } from 'react';
```

Comment:
react is already imported on line one. We should be able to find an eslint plugin to handle this

Reply:
https://eslint.org/docs/latest/rules/no-duplicate-imports

Reply:
This is intentional. We have a rule to separate type imports from code imports.

It’s even written on the COLLABORATOR_GUIDE if you give it a read 🙃

Reply:
oh i missed the type - thanks!